java/squeek/veganoption/blocks/BlockComposter.java

```diff
@@ -125,4 +133,18 @@ public boolean isFullCube(IBlockState state)
 	{
 		return false;
 	}
+
+	@Override
+	public void addProbeInfo(ProbeMode mode, IProbeInfo probeInfo, EntityPlayer player, World world, IBlockState blockState, IProbeHitData data)
+	{
+		TileEntityComposter composter = (TileEntityComposter) world.getTileEntity(data.getPos());
+		if (composter == null)
+			return;
+
+		if (composter.isComposting())
+			probeInfo.text(String.format("%s : %d%%", LangHelper.translate("waila.composter.composting"), (int) (composter.getCompostingPercent() * 100F)))
+				.text(String.format("%s : %.0f" + ProviderComposter.DEGREE_SYMBOL + "C", LangHelper.translate("waila.composter.temperature"), composter.getCompostTemperature()));
```

Comment:
This `ProviderComposter.DEGREE_SYMBOL` reference might cause problems when Waila is not installed (ProviderComposter is normally only loaded when Waila is installed, so the Waila interface is assumed to exist).

Could just move the DEGREE_SYMBOL constant to BlockComposter and use `BlockComposter.DEGREE_SYMBOL` in the Waila file instead.